style.css

```diff
@@ -0,0 +1,1067 @@
+body {
+  font-family: "Calibri Bold", AvenirNext, Avenir, -apple-system, BlinkMacSystemFont, "Roboto Slab", "Droid Serif", "Segoe UI", Oxygen-Sans, Ubuntu, Cantarell, Georgia, serif;
+  box-sizing: border-box;
+  overflow-x: hidden;
+  margin: 0;
+  padding: 0;
+  max-width: 1440px;
+}
+
+.header,
+.main,
+.footer-main {
+  max-width: 1440px;
+}
+
+.nav-container {
+  display: flex;
+  align-items: flex-start;
+  text-transform: capitalize;
+  padding-top: 5px;
+  padding-bottom: 25px;
+  padding-left: 15px;
+  padding-right: 15px;
```

Comment:
**Require shorthand properties:**  The properties padding-top, padding-bottom, padding-left, padding-right can be replaced by padding.